packages/driver/package.json

```diff
@@ -16,6 +16,7 @@
     "@babel/code-frame": "7.8.3",
     "@cypress/sinon-chai": "2.9.1",
     "@cypress/unique-selector": "0.4.4",
+    "@cypress/webpack-dev-server": "0.0.0-development",
```

Comment:
Why was this dependency added?

Reply:
I was getting local typesript errors because we use it in our cypress.config.js file: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress.config.ts#L2